### PR TITLE
instructor editing questions: Make the 'responses will be deleted' message harder to miss #4566

### LINKED ISSUE #4566: instructor editing questions: Make the 'responses will be deleted' message harder to miss
Currently, it looks like a regular message and users might click 'OK' without reading it.

### PR DESCRIPTION
**Pull request by xeia**
Fixes #4566

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@xeia but did you fix the HTML files manually?

**Comment by xeia** [comment]
@wkurniawan07 err i edited the test HTML after each run of all tests..

**Comment by wkurniawan07** [comment]
@xeia let me rephrase, then: did you use [this technique](https://github.com/TEAMMATES/teammates/blob/master/devdocs/godmode.md) to fix the HTML files?

**Comment by xeia** [comment]
Ah sorry my bad, I'll revert the changes and follow the technique instead

**Comment by wkurniawan07** [comment]
Setting it back to ongoing, tag me when it's ready for review!

**Comment by xeia** [comment]
@wkurniawan07 Should be ready now

**Comment by wkurniawan07** [comment]
This one looks good :+1:

**Comment by wkurniawan07** [comment]
@xeia can you update this branch and do any necessary fixes so that I can merge it?

**Comment by wkurniawan07** [comment]
@xeia you're lucky I'm awake to watch Champions League final... :stuck_out_tongue:

**Comment by xeia** [comment]
@wkurniawan07 hahaha! sorry for doing this so late at night and thanks alot!
